plugin/plugin.go

```diff
@@ -21,6 +21,7 @@ const (
 type Args struct {
 	Level             string `envconfig:"PLUGIN_LOG_LEVEL"`
 	RoutingKey        string `envconfig:"PLUGIN_ROUTING_KEY"`
```

Comment:
remove all refs of RoutingKey and update it to IntegrationKey

Reply:
@Ompragash, In Jenkins plugin it is routing key for trigger incident and incident key for change event, That's why I kept two separate variables for each one of them. please find the below attached snip from jenkins plugin for your reference.
![image](https://github.com/user-attachments/assets/5459decd-7509-4df2-9450-c6e68d4a325f)